readme.md

```diff
@@ -97,7 +97,7 @@
 
 ## Get Smart
 
-### [Annotated ECMAScript 5.1](http://es5.github.com/)
+### [Annotated ECMAScript 5.1](http://es5.github.io/)
```

Comment:
The link is currently broken, as it is. While this link does work, the https version should be preferred, in my opinion.

https://es5.github.io/